clang/lib/DPCT/APINamesMathRewrite.inc

```diff
@@ -515,9 +515,245 @@ MATH_API_REWRITER_HOST_DEVICE(
 MATH_API_REWRITER_DEVICE(
     "saturate",
     MATH_API_DEVICE_NODES(
-        CALL_FACTORY_ENTRY("saturate",
-                           CALL(MapNames::getClNamespace(false, true) +
-                                    "clamp<float>",
-                                ARG(0), ARG("0.0f"), ARG("1.0f"))),
-        EMPTY_FACTORY_ENTRY("saturate"), EMPTY_FACTORY_ENTRY("saturate"),
-        EMPTY_FACTORY_ENTRY("saturate")))
+    CALL_FACTORY_ENTRY("saturate",
+                CALL(MapNames::getClNamespace(false, true) +
+                        "clamp<float>",
+                    ARG(0), ARG("0.0f"), ARG("1.0f"))),
+    EMPTY_FACTORY_ENTRY("saturate"), EMPTY_FACTORY_ENTRY("saturate"),
+    EMPTY_FACTORY_ENTRY("saturate")))
+
+MATH_API_REWRITER_DEVICE(
```

Comment:
Should better put these API rewriter in the corresponding section according to CUDA doc.